patterns/photo-blog-single-post-template.php

```diff
@@ -53,15 +53,15 @@
 						<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
 						<div class="wp-block-group">
 							<!-- wp:paragraph {"fontSize":"small"} -->
-							<p class="has-small-font-size">Categories:</p>
+							<p class="has-small-font-size"><?php esc_html_e( 'Categories', 'twentytwentyfive' ); ?>:</p>
```

Comment:
Let's keep the `:` to match the design.

Reply:
@juanfra I'm unsure about this one. It looks like the `:` is added after the `Catgories` and `Tagged` strings in the PR:

https://github.com/WordPress/twentytwentyfive/pull/355/files#diff-d8a2a54e1164459ea5b389207d5c9493b8f09d3473f5aeb4f98563a72376a9bf

Happy to fix if needed, but it seems like the change is already present.

Reply:
Thanks, I meant inside the translation. We can leave it as it is right now, though.